src/components/home/UserInput.vue

```diff
@@ -219,7 +219,7 @@ export default Vue.extend({
       }
     },
     parseContent(fileName: string, content: string) {
-      const extension = /\.csv$/.test(fileName) ? '.csv' : '.bed'
+      const extension = '.csv'
```

Comment:
Keep allowing bed files. Just don't mention it in the label and tooltips. The functionality should not be removed, just hidden.

Reply:
done